src/main/java/org/acra/builder/ReportExecutor.java

```diff
@@ -276,9 +292,11 @@ public void run() {
                 }
                 lastActivityManager.clearLastActivity();
             }
-
-            android.os.Process.killProcess(android.os.Process.myPid());
-            System.exit(10);
+            //prevent process kill if a debugger is attached, as this would kill the whole application
+            if (!Debug.isDebuggerConnected()) {
```

Comment:
We can not hand the exception to the defaultExceptionHandler. It will also kill the process. Clearing the last activity is okay though (and as this is the only thing we can do to minimize inconsistent states I think this should definetly be done).